components/brave_welcome_ui/containers/screens/shieldsBox.tsx

```diff
@@ -30,9 +32,17 @@ export default class ShieldsBox extends React.PureComponent<Props> {
       >
         <WelcomeShieldsImage />
         <Title>{getLocale('privacyTitle')}</Title>
-        <Paragraph
-          dangerouslySetInnerHTML={{ __html: getLocale('privacyDesc') }}
-        />
+        <Paragraph>
+          {text[0]}
+          <a href='https://brave.com/p3a'
+             target='_blank'
+             rel='noopener noreferrer'>{text[1]}</a>
```

Comment:
tslint will probably want the `>` on a new line.

Reply:
tslint errors fixed, thanks!